.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false
 language: go
 
 go:
-- 1.5
+- 1.6.2
```

Comment:
Wouldn't be better to continue to test against go `1.5` and `1.6` ? People might continue to build it by hand on `go 1.5`.

```
go:
- 1.5.4
- 1.6.2
```